src/app/lorax/directives/modal-issue.tpl.html

```diff
@@ -0,0 +1,28 @@
+<article class="modal modal--fade-show modal-issue" ng-show="modalIssue.open" >
+  <a href="#" class="modal__close modal__close-absolute icon-close" data-lorax-prevent-default ng-click="modalIssue.closeModal()">Close</a>
+  <div class="container">
+    <h1>{{modalIssue.issue.getName()}}</h1>
```

Comment:
Yeah, I was wondering about that as well, I would think that one should not have more than one, to assist with document structure but, with the changes in HTML5 on can technically have more than one now, especially in different `<section>` and `<article>` blocks but, still not sure about this one. I will have a look at the overall document structure, which is a problem I identified before, and see how this impacts it. For now, I left it as it was.